shell/platform/darwin/macos/framework/Source/FlutterPlatformNodeDelegateMac.mm

```diff
@@ -35,6 +35,17 @@
   NSCAssert(ax_platform_node_, @"Failed to create platform node.");
 }
 
+void FlutterPlatformNodeDelegateMac::NodeDataChanged(const ui::AXNodeData& old_node_data,
+                                                     const ui::AXNodeData& new_node_data) {
```

Comment:
I wonder if we should add a comment in `Init` that if we add any other special cases, to ensure they're covered in both Init and here. Maybe obvious enough since they're side-by-side.

Reply:
My gut feeling is that the methods being side by side should be enough.